style/modules/pressModule.css

```diff
@@ -0,0 +1,37 @@
+.pressModule {
+    display: flex;
+    /* overflow: hidden; */
+    background: #f5f5f5;
+    /* width: 1080px; */
+    height: 250px;
+    width: 1200px;
```

Comment:
to make this scale to the browser width, we'll want `width: 100%` rather than a pixel value. (I tested this and it does work -- the paragraph text won't stretch the container as long as there's *some* kind of width set)